set.mm

```diff
@@ -184668,7 +184747,8 @@ their least common multiple ( ~ df-lcmf ) are provided.  Both definitions are
 
     $( Two positive integers are not coprime, i.e. there is an integer greater
        than 1 which divides both integers, iff their greatest common divisor is
-       not 1.  (Contributed by AV, 9-Aug-2020.) $)
+       not 1.  See ~ ncoprmgcdne1bgd for a more general version.  (Contributed
+       by AV, 9-Aug-2020.) $)
     ncoprmgcdne1b $p |- ( ( A e. NN /\ B e. NN )
                           -> ( E. i e. ( ZZ>= ` 2 ) ( i || A /\ i || B )
                                <-> ( A gcd B ) =/= 1 ) ) $=
```

Comment:
True for any $A, B \in \mathbb{Z}$.

Reply:
I'll leave this (also) to a later pr since a few parent theorems would also need to be generalized